mix.exs

```diff
@@ -76,7 +76,8 @@ defmodule Skate.MixProject do
       {:postgrex, "~> 0.15"},
       {:ex_aws_rds, "~> 2.0.2"},
       {:fast_local_datetime, "~> 1.0"},
-      {:ex_machina, "~> 2.7.0", only: :test}
+      {:ex_machina, "~> 2.7.0", only: :test},
+      {:logster, "~> 1.0"}
```

Comment:
Not related to this line specifically, but…

`logger` isn't defined here in the `deps` list, but it is still above in the list of `extra_applications`. Is it still required to be there for other libraries or Phoenix or something?

Reply:
My understanding is that Logster is just a plug that calls Logger with a nicer format than the default, so we still want that running.

Reply:
It looks like the API [also leaves the `:logger` process running](https://github.com/mbta/api/blob/master/apps/api_web/mix.exs#L28), for a point of comparison, so I feel pretty good about merging this as-is.